srcpkgs/textadept/template

```diff
@@ -0,0 +1,49 @@
+# Template file for 'textadept'
+pkgname=textadept
+version=10.6
+revision=1
+archs="i686 x86_64"
+wrksrc=textadept-textadept_${version}
+hostmakedepends="autoconf automake gettext libtool clang wget unzip darcs pkg-config glib-devel ncurses"
```

Comment:
Void is build with gcc and clang isn't used during build anyway. gettext is available by default. Ideally, deps should be added to distfiles for caching, then wget, unzip, maybe darcs won't be needed too.

Reply:
Some files cannot be downloaded when added to distfiles. e.g. I noticed it runs `wget -O bombay.zip http://foicica.com/hg/bombay/archive/tip.zip`. So I added `http://foicica.com/hg/bombay/archive/tip.zip>bombay.zip` to distfiles and it complains like this:

```
=> textadept-10.6_1: fetching distfile 'bombay.zip'...
http://foicica.com/hg/bombay/archive/tip.zip: (null)
=> ERROR: textadept-10.6_1: failed to fetch bombay.zip.
```

But I can access it fine on a browser, but it downloads `bombay-3852895f5313.zip`. Do you know if xbps can handle redirection?

Reply:
Use these addresses then. We should not use unversioned sources anyway.

One address per line please, same like checksum.

Files are downloaded by `make deps` anyway. You need to copy into build dir like libreoffice do.